docs/addConverter.md

```diff
@@ -44,7 +44,7 @@
   ```
   @codepen
 
-  @param {String} converterName The name of the converter helper.
+  @param {String|Object} converterName The name of the converter helper.	An object to register multiple converters at once.
```

Comment:
"The name of the converter helper or an object to register multiple converters at once"

Reply:
Done!